internal/credentials/command.go

```diff
@@ -17,16 +17,15 @@ var (
 // Command is the base command for managing profiles
 var Command = &cobra.Command{
 	Use:   "profiles",
-	Short: "Subcommands to mange the credential profiles for this tool",
+	Short: "Manage the credential profiles for this tool",
```

Comment:
Not a blocker, similar to other suggestion...
```suggestion
	Short: "Manage the credential profiles for the New Relic CLI",
```

Reply:
👍